include/nccl_ofi_param.h

```diff
@@ -249,7 +249,7 @@ OFI_NCCL_PARAM_INT(disable_dmabuf, "DISABLE_DMABUF", 0);
 /*
  * Messages sized larger than this threshold will be striped across multiple rails
  */
-OFI_NCCL_PARAM_UINT(min_stripe_size, "MIN_STRIPE_SIZE", (64 * 1024));
```

Comment:
Split this into its own commit, it deserves its own headline when you run `git log --oneline`.

Reply:
And you need a bit of why when you write the commit message for this change.

Reply:
+1. As evident from `We also` in commit message, you are doing multiple logical changes in single commit rather than one.

Reply:
Done. I have split the min_stripe_size from 64K to 128k to a separate commit.